tensorflow_probability/examples/jupyter_notebooks/Probabilistic_Principal_Component_Analysis.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
If you elect to check this in as a separate notebook, we should rename the file to something like `Probabilistic_PCA_HMC.ipynb` to distinguish from the VI example.